NitroxModel/Helper/Reflect.cs

```diff
@@ -1,128 +1,137 @@
-﻿using System;
+﻿extern alias JB;
```

Comment:
What is the reason you include `JetBrains.Annotations` here?

Reply:
No reason anymore. Forgot to remove it.